packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -64,7 +64,7 @@ import { Phase, prepareVault } from './vault.js';
 
 const { details: X, Fail } = assert;
 
-const trace = makeTracer('VM', false);
+const trace = makeTracer('VM');
```

Comment:
I'm afraid this bit is going to get worn out from being turned off and on so much :)

Reply:
same! I see if snapping off. If only we had a way to configure debugging at runtime. Imagine!